### DISCUSSION TITLE
How does GitHub ensure security for user authentication?

### DISCUSSION THREAD

**Discussion by maksymhu** [community/community#114637] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How does GitHub ensure security for user authentication?

**Comment by ethanwater** [comment]
They use similar techniques to other major companies like: 2FA, Tokens, HTTPS encryption, etcetera.

This link should help you out: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/about-authentication-to-github

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @olehrab ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
